src/irr/asset/CGraphicsPipelineLoaderMTL.cpp

```diff
@@ -381,7 +379,7 @@ void main()
     case 9:
     {
         float VdotN = dot(interaction.N, interaction.V.dir);
```

Comment:
you can get that from isotropic paramters I think?

Reply:
thats the whole point of it that i cant get isotropic parameters in main()
https://github.com/buildaworldnet/IrrlichtBAW/pull/410#discussion_r369297463

Reply:
yeah, you're right about the comment, although I should make a note to myself to optimize after I merge (`VdotN` could be stored in the surface interaction, since its the same for every light)